processing/src/main/java/io/druid/query/extraction/ExtractionFn.java

```diff
@@ -31,7 +31,8 @@
     @JsonSubTypes.Type(name = "javascript", value = JavascriptExtractionFn.class),
     @JsonSubTypes.Type(name = "timeFormat", value = TimeFormatExtractionFn.class),
     @JsonSubTypes.Type(name = "identity", value = IdentityExtractionFn.class),
-    @JsonSubTypes.Type(name = "lookup", value = LookupExtractionFn.class)
+    @JsonSubTypes.Type(name = "lookup", value = LookupExtractionFn.class),
+    @JsonSubTypes.Type(name = "concat", value = ConcatExtractionFn.class)
```

Comment:
agree wth @himanshug. I think "StringFormat" is a better name for this extraction fn